docs/roadmap.md

```diff
@@ -0,0 +1,24 @@
+# Roadmap
```

Comment:
Is it worth adding https://github.com/xtuc/js-webassembly-interpreter/issues/152 to the roadmap? I'd worry that without this fix this project is going to need to make a breaking change just to rename parts of the API

Reply:
Yes good point, i'll add it.